mod/state-transition/pkg/core/state_processor.go

```diff
@@ -437,82 +432,3 @@ func (sp *StateProcessor[
 	)
 	return st.SetLatestBlockHeader(lbh)
 }
-
-// getAttestationDeltas as defined in the Ethereum 2.0 specification.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_attestation_deltas
-//
-//nolint:lll
-func (sp *StateProcessor[
-	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, _, _, _, _, _,
-]) getAttestationDeltas(
-	st BeaconStateT,
-) ([]math.Gwei, []math.Gwei, error) {
-	// TODO: implement this function forreal
-	validators, err := st.GetValidators()
-	if err != nil {
-		return nil, nil, err
-	}
-	placeholder := make([]math.Gwei, len(validators))
-	return placeholder, placeholder, nil
```

Comment:
we load the full validator list to do absolutely nothing with it. Better avoid this until we explicitly handle slashing